runtime/commonMain/src/kotlinx/serialization/builtins/CollectionSerializers.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.builtins
+
+import kotlinx.serialization.*
+import kotlinx.serialization.internal.*
+
+/**
+ * Creates a serializer for [`List<T>`][List] for the given serializer of type [T].
+ */
+public val <T> KSerializer<T>.list: KSerializer<List<T>>
```

Comment:
I'd say that (apart from `.nullable`) `.list` and `.set` extensions are neither readable nor conventional.
Now, when we have a replacement, WDYT about deprecating it?